ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -2,18 +2,22 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::metrics::{
-    ERROR_COUNT, LATEST_PROCESSED_VERSION, PROCESSED_BATCH_SIZE, PROCESSED_LATENCY_IN_SECS,
-    PROCESSED_VERSIONS_COUNT,
+    ERROR_COUNT, LATEST_PROCESSED_VERSION as LATEST_PROCESSED_VERSION_OLD, PROCESSED_BATCH_SIZE,
```

Comment:
why is it renamed to OLD? Need to add some context on whether we plan to deprecate etc.